src/collections/map-of-the-week/mon-apr-01-2024-13-00-00-gmt-0600-central-standard-time.md

```diff
@@ -0,0 +1,5 @@
+---
+mapId: 3a868
+review: Genius work in vocal mixing, mapping, and lighting to present a map with a variety of experiences; from speed, to tech, and fit, with two lowers for accessibility, all of which completed with high standards.

```

Comment:
You just copied it, but isn't that missing a word before completed? Like "is completed" or "are completed"?